### DISCUSSION TITLE
Unable to create project from Github repository

### DISCUSSION THREAD

**Discussion by radenpioneer** [vercel/community#977] [Help] [answered]
I am unable to import Git repository from Github, everytime I try to connect to Github using "Continue with Github" button, it opens Github Unicorn error.

![image](https://user-images.githubusercontent.com/7355835/197335842-b63d8dff-18ca-4891-9abc-c8e854f49ef0.png)

Both Github status and Vercel status shows that all systems are operational.

![image](https://user-images.githubusercontent.com/7355835/197335962-1a854d68-b20e-4ac3-9777-f1cadac097f1.png)

![image](https://user-images.githubusercontent.com/7355835/197335975-4b52ec92-958b-4633-9b87-d6892244cf64.png)

**Comment by mkaufmann84** [comment]
I am also having this issue.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out! This error is actually coming from GitHub. The most common cause I have seen is belonging to an org with a large number of repos, such as one used for school coursework, which can be solved by leaving that org. You should [submit a bug report to GitHub](https://support.github.com/contact/bug-report) so they know what's happening. They will have access to their error logs to help find the source of the problem for you and get you unblocked quickly.
